pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -528,8 +547,21 @@ message ContainerConfig {
     //     prefix ::= DNS_SUBDOMAIN
     //     name ::= DNS_LABEL
     map<string, string> labels = 9;
-    // Annotations is an unstructured key value map that may be set by external
-    // tools to store and retrieve arbitrary metadata.
+    // Unstructured key-value map that may be used by the kubelet to store and
+    // retrieve arbitrary metadata.
+    //
+    // Annotations MUST NOT be altered by the runtime; the annotations stored
+    // here MUST be returned in the ContainerStatus associated with the container
+    // this ContainerConfig creates.
+    //
+    // In general, in order to preserve a well-defined interface between the
+    // kubelet and the container runtime, annotations SHOULD NOT influence
+    // runtime behaviour.
+    //
+    // Annotations can be useful for runtime authors to experiment with
+    // new features that are opaque to the Kubernetes APIs (both user-facing
```

Comment:
@jonboulle It is not true because

- there is no way to set container annotations by kubernetes API
- `Annotations MUST NOT be altered by the runtime` as above said

Since container annotations couldn't be changed, then how could it help to experiment with new features?